.github/workflows/build-and-benchmark-x86.yml

```diff
@@ -0,0 +1,47 @@
+name: build and benchmark on x86-64
+on: [push, pull_request]
+jobs:
+    benchmark-on-self-hosted-x86-64:
+        runs-on: [self-hosted, linux, X64]
+        steps:
+            - name: git checkout
+              uses: actions/checkout@v4
+            - name: release build this PR
+              run:  nice cargo build --release
+            - name: release build main branch
+              run: |
+                git fetch --depth 1 origin/main && git checkout origin/main
+                nice cargo build --release --target-dir target.main
+            - name: benchmark on chimera 8-bit test data
+              run: |
+                mkdir -p `dirname $LOCAL_FILE`
```

Comment:
```suggestion
                mkdir -p $(dirname $LOCAL_FILE`)
```

Reply:
It makes no difference which syntax is used here.